datacenter/dtr/2.2/guides/admin/configure/jobrunner.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Jobrunner
+description: Learn about the inner-workings of the jobrunner container in the DTR workflow.
+keywords: docker, job, runner
+---
+
+The jobrunner container is a DTR mechanism that:
+
+1. Consumes jobs from a cluster-wide jobs queue 
+2. Performs the work of the given action
+
+There is one jobrunner container per replica. 
+
+## Jobrunner Workflow
+The following diagram depicts the behavior of the jobrunner:
```

Comment:
I added a paragraph below it describing job scheduling. We'll also put in a diagram of how it works to make it more clear.